api/pkg/apis/v1alpha1/providers/target/k8s/k8s.go

```diff
@@ -288,6 +288,9 @@ func (i *K8sTargetProvider) getDeployment(ctx context.Context, namespace string,
 		log.ErrorfCtx(ctx, "  P (K8s Target): getDeployment %s failed - %s", name, err.Error())
 		return nil, err
 	}
+	if deployment.DeletionTimestamp != nil {
+		return nil, nil
```

Comment:
Add one line log to record this deleting deployment.

Reply:
added